docs/_config.yml

```diff
@@ -12,6 +12,8 @@ ALLUXIO_MASTER_VERSION_SHORT: master
 # For release candidates and release branches, this should be the major Alluxio version, e.g. both 1.5.0 and 1.5.0-RC1 should use "1.5"
 # For master branch, this should be "master"
 ALLUXIO_MAJOR_VERSION: master
+# The version string must match the version string part of the client jar path
+ALLUXIO_VERSION_STRING: 2.0.0-SNAPSHOT
```

Comment:
Could we avoid adding this new variable?

Reply:
I don't have a good solution. The problem here is `master` is not a valid string for the places you want to use it. It is quite painful to maintain a bunch of variables in `_config.yml` though.

Reply:
yep, that's the trouble I had. but since users often copy paste, i would like to reduce the manual steps of replacing strings from the doc

Reply:
@calvinjia reverting this here and opening a separate PR to ease cherry-pick